timify-api/models/episode.js

```diff
@@ -1,62 +1,59 @@
 import { DataTypes } from 'sequelize';
 import { sequelize } from '../database.js';
-import { PodcastSeries } from './podcastseries.js';
 
 export const Episode = sequelize.define('Episode', {
     uuid: {
-        type: DataTypes.UUID,
-        defaultValue: DataTypes.UUIDV4,
-        primaryKey: true
-      },
+      type: DataTypes.UUID,
+      defaultValue: DataTypes.UUIDV4,
+      primaryKey: true
+    },
+    podcastUuid: {
+      type: DataTypes.UUID,
+      allowNull: false,
+      references: {
+        model: 'Podcast',
+        key: 'uuid'
+      }
+    },
+    hash: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
     name: {
-        type: DataTypes.STRING,
-        allowNull: false
+      type: DataTypes.STRING,
+      allowNull: false
     },
     description: {
-        type: DataTypes.TEXT,
-        allowNull: false
+      type: DataTypes.TEXT,
+      allowNull: false
     },
     imageUrl: {
-        type: DataTypes.STRING,
-        allowNull: false
+      type: DataTypes.STRING,
+      allowNull: false
     },
     datePublished: {
-        type: DataTypes.INTEGER,
-        allowNull: false
+      type: DataTypes.DATE,
+      allowNull: false
     },
-    hash: {
-        type: DataTypes.STRING,
-        allowNull: false
+    subtitle: {
+      type: DataTypes.STRING
     },
     audioUrl: {
-        type: DataTypes.STRING,
-        allowNull: false
+      type: DataTypes.STRING
     },
     fileLength: {
-        type: DataTypes.INTEGER,
-        allowNull: false
+      type: DataTypes.INTEGER
     },
     fileType: {
-        type: DataTypes.STRING,
-        allowNull: false
+      type: DataTypes.STRING
     },
     duration: {
-        type: DataTypes.INTEGER,
-        allowNull: false
+      type: DataTypes.INTEGER
     },
     episodeNumber: {
-        type: DataTypes.INTEGER,
-        allowNull: false
-    },
-    podcastSeriesUuid: {
-        type: DataTypes.UUID,
-        references: {
-          model: PodcastSeries,
-          key: 'uuid'
-        }
-    }
-    
-})
+      type: DataTypes.INTEGER
+    },
+  });
+  
 
-PodcastEpisode.belongsTo(PodcastSeries, { foreignKey: 'podcastSeriesUuid', as: 'podcastSeries' });
```

Comment:
Are we gonna deprecate whole podcast series? What's the relationship between podcast and podcast series? From my understanding, episode belongs to podcast, podcast belongs to podcast series?

Reply:
I honestly don't know how you can still see that because i deleted it. i took out podcast series and just made a direct relationship between podcast and episodes. this is the current relationship: 

Podcast.hasMany(Episode, { foreignKey: 'episodeUuid' });
Episode.belongsTo(Podcast, { foreignKey: 'podcastUuid' });